src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -23146,6 +23146,94 @@ static void M(C c, string? p1, string? p2)
             CompileAndVerify(comp2);
         }
 
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

Please add a test with attribute on a local function. It will fail for now, but will pass sometime soon ;-)